<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>homepage</class>
 <widget class="QMainWindow" name="homepage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1095</width>
    <height>815</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>20</width>
      <height>661</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="friendsbox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>281</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#friendsbox{

background-color: rgb(243, 243, 243);
   border-radius: 10px;
}</string>
    </property>
    <property name="title">
     <string>Friends List</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QListWidget" name="listoffriends"/>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>350</y>
      <width>641</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="topPanel" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#topPanel {
	
	background-color: rgb(243, 243, 243);
}</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>431</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QWidget" name="LoginIcon" native="true">
        <property name="styleSheet">
         <string notr="true">#LoginIcon {
	border-image: url(:/images/logo.png);
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Ubuntu Condensed</family>
          <pointsize>48</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Super Message</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>490</y>
      <width>641</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="Profilepicbox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>281</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#Profilepicbox{

background-color: rgb(243, 243, 243);
   border-radius: 10px;
}
</string>
    </property>
    <property name="title">
     <string>Profile Picture</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QLabel" name="ProfilePic">
       <property name="styleSheet">
        <string notr="true">#ProfilePic{

background-image: url(:/images/emptyprofilepic.png);
height: 20px;
width: 20px;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="src.qrc">:/images/emptyprofilepic.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="EditProfilePic">
       <property name="text">
        <string>Upload New Picture</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>370</y>
      <width>641</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#groupBox {
	background-color: rgb(255, 255, 255);
 border-radius: 10px;
}</string>
    </property>
    <property name="title">
     <string>Result</string>
    </property>
    <widget class="QLabel" name="searchusernameresult">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Username:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Email:</string>
     </property>
    </widget>
    <widget class="QLabel" name="searchemailresult">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Addfriend">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>60</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Friend</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ViewHistory">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>60</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>View History</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SendMessage">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>60</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Send Message</string>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_5">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>510</y>
      <width>641</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#groupBox_2 {
	background-color: rgb(255, 255, 255);
 border-radius: 10px;
}</string>
    </property>
    <property name="title">
     <string>Stats</string>
    </property>
    <widget class="QPushButton" name="ViewAllHistory">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>30</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>View All History</string>
     </property>
    </widget>
    <widget class="QPushButton" name="StatisticalReport">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>70</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Statistical Report</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>240</y>
      <width>641</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#groupBox_3 {
	background-color: rgb(255, 255, 255);
 border-radius: 10px;
}</string>
    </property>
    <property name="title">
     <string>Search Member</string>
    </property>
    <widget class="QPushButton" name="Searchbutton">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>50</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="SearchBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>441</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Memberbox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>110</y>
      <width>641</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#Memberbox{

	background-color: rgb(114, 159, 207);
   border-radius: 10px;
}
</string>
    </property>
    <property name="title">
     <string>Member: Logged in</string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Username:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Email:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Mobile:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>201</width>
       <height>74</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="User1_Username">
        <property name="text">
         <string>Username</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="User1_Email">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="User1_Mobile">
        <property name="text">
         <string>Mobile</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="EditProfilebutton">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>50</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Edit Profile</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1095</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
